Hashids/Hashids.swift

```diff
@@ -305,7 +307,7 @@ func transform<T: CollectionType where T.Generator.Element: Equatable>(a: T, _ b
 }
 
 func unique<T: CollectionType where T.Generator.Element: Equatable>(a: T) -> [T.Generator.Element] {
-    return transform(a, a) { (var c, a, b, e) in
+    return transform(a, a) { (inout c: [T.Generator.Element], a, b, e) in
```

Comment:
gotcha, I can just remove the var, I didn't realize it was already recognizing it as an `inout`